source/index.js

```diff
@@ -19,6 +19,11 @@ var likely = function (node, options) {
         widget.update(fullOptions);
     }
     else {
+        var canonicalUrl = utils.getCanonicalUrl();
+
+        if (canonicalUrl) {
+            fullOptions.url = canonicalUrl;
+        }
```

Comment:
Doesn’t this override the url passed from the options? We want the url in the options be more specific than the canonical url, and the canonical url be more specific than the page url.

Also, I’d better move this to https://github.com/ilyabirman/Likely/blob/master/source/index.js#L50, where the page URL is extracted. There can only be one canonical URL per page, so it belongs rather to Likely defaults than to its constructor.